Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientOutputs.idl

```diff
@@ -29,10 +29,20 @@
     boolean rk;
 };
 
+[
+    Conditional=WEB_AUTHN,
+    JSGenerateToJSObject,
+] dictionary LargeBlobOutputs {
```

Comment:
Seems to be called AuthenticationExtensionsLargeBlobOutputs in the spec?

Reply:
Ditto